crates/astria-core/src/primitive/v1/mod.rs

```diff
@@ -22,6 +22,8 @@ pub const ADDRESS_LEN: usize = 20;
 
 pub const ROLLUP_ID_LEN: usize = 32;
 
+pub const TRANSACTION_ID_LENGTH: usize = 32;
```

Comment:
Folllow the convention of the constant right above it.
```suggestion
pub const TRANSACTION_ID_LEN: usize = 32;
```